### PR TITLE
homepage as logged-in user completed

### PR DESCRIPTION
**Pull request by DrewGodeke**
Completed the homepage view and CSS for a logged-in user. We can still adjust the font, size, or color of our welcome message; I think it's a bit difficult to see.
Screenshot 2024-05-25 190806

### CODE REVIEW

**Comment by lauradumi** [comment]
look good Andrew. I would try Increasing the size of "Welcome" and maybe bringing down the avatar as to leave more space for the welcome message.

**Review by lauradumi — DISMISSED** [review]
You should change your avatar link to the one of the images that Gabriel added to our assets.

**Review by DrewGodeke — COMMENTED** [review]
ignore

**Comment by DrewGodeke** [comment]
Screenshot 2024-05-28 211919

File: app/views/pages/home.html.erb
Comment (DrewGodeke) [inline app/views/pages/home.html.erb:1]: I only added basic styling since it wont be shown at all  during the demo.


Screenshot 2024-05-30 153812

**Review by lauradumi — DISMISSED** [review]
lgtm

**Review by lauradumi — APPROVED** [review]

**Pull request merged**